src/libcore/num/mod.rs

```diff
@@ -212,11 +212,20 @@ pub mod diy_float;
 
 mod wrapping;
 
+macro_rules! usize_isize_to_xe_bytes_doc {
+    () => {"
+
+This function returns an array of length 2, 4 or 8 bytes depending
```

Comment:
It might be a good idea to add a bold `**NOTE**:` -- a (partially) wrong function signature being shown is pretty important.